### PR TITLE
Review cleanup

### PR DESCRIPTION
**Pull request by wipfli**

### CODE REVIEW

File: src/source/tile.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import {uniqueId, parseCacheControl} from '../util/util';
 import {deserialize as deserializeBucket} from '../data/bucket';
-import '../data/feature_index';
-import type FeatureIndex from '../data/feature_index';
+import FeatureIndex from '../data/feature_index';
```
Comment (HarelM) [inline src/source/tile.ts:3]: This is needed. You'll need this reverted in order for the tests to pass and the library to behave as expected.

File: src/ui/control/scale_control.ts [hunk 2]
```diff
@@ -128,7 +128,7 @@ function getDecimalRoundNum(d) {
 }
 
 function getRoundNum(num) {
-    const pow10 = Math.pow(10, ((`${Math.floor(num)}`)).length - 1);
+    const pow10 = Math.pow(10, (`${Math.floor(num)}`).length - 1);
```
Comment (HarelM) [inline src/ui/control/scale_control.ts:131]: First time I'm seeing this - wouldn't it be easier to just do `% 10` or something? In any case it's not related to this PR...

**Comment by HarelM** [comment]
Besides a single place that the change was not correct everything else looks good.

File: src/source/tile.ts (hunk 1 above)
Comment (wipfli) [reply src/source/tile.ts:3]: Will revert

File: src/ui/control/scale_control.ts (hunk 2 above)
Comment (wipfli) [reply src/ui/control/scale_control.ts:131]: Yes, very strange indeed.

**Pull request merged**